Jenkinsfile.publish

```diff
@@ -7,6 +7,17 @@ properties([
     [$class: 'JobRestrictionProperty'], 
     pipelineTriggers([pollSCM('H 5 * * * ')])
 ])
+import java.text.SimpleDateFormat
```

Comment:
would rather prefer to have a separate jenkins job, that just trigger with cron

Reply:
@chrmod Done. Nightly job: http://magrathea:8080/job/browser-ios/job/nightly/
Trigger on tags Multibranch: http://magrathea:8080/job/browser-ios/job/nightly-multibranch/